File: wbbackend/src/main/java/com/wb/backend/entidades/Servico.java
package com.wb.backend.entidades;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import org.springframework.hateoas.RepresentationModel;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class Servico extends RepresentationModel<Produto> {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	
	@Column
	private String nome;

	@Column
	private String descricao;

	@Column
	private String preco;

    @Column
	private String duracao;

	public Servico(){
	}

	public Servico(String nome, String descricao, String preco, String duracao){
		this.nome = nome;
		this.descricao = descricao;
		this.preco = preco;
		this.duracao = duracao;
	}
}

File: atv01/src/modelo/servico.ts
export default class Servico {
    public nome: string
    public descricao: string
    public duracao: string
    public preco: string
    public genero: string

    constructor(nome: string, descricao: string, duracao: string, preco: string, genero: string){
        this.nome = nome
        this.descricao = descricao
        this.duracao = duracao
        this.preco = preco
        this.genero = genero
    }
    public atualizarDados(nome: string, descricao: string, duracao: string, preco: string, genero: string): void {
        this.nome = nome;
        this.descricao = descricao
        this.duracao = duracao
        this.preco = preco
        this.genero = genero
    }

}